pkg/services/alerting/conditions/query_test.go

```diff
@@ -55,7 +55,7 @@ func TestQueryCondition(t *testing.T) {
 
 			Convey("should fire when avg is above 100 on dataframe", func() {
 				ctx.frame = data.NewFrame("",
-					data.NewField("time", nil, []time.Time{time.Now()}),
+					data.NewField("time", nil, []time.Time{time.Now(), time.Now()}),
```

Comment:
This was an "invalid data frame" before since the length of time and value fields didn't add up. I'm not really sure why this wasn't caught earlier since in my opinion we are doing the same thing as before.